tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
```

Comment:
Just some pytest and tox configuration, so that executing `pytest` or `tox` on top level will know where to look for the tests (and in case of tox, with which interpreters it should be tested).